docs/development/modules.md

```diff
@@ -324,6 +324,25 @@ public class MyTransformModule implements DruidModule {
 }
 ```
 
+### Adding your own custom pluggable Coordinator Duty
```

Comment:
Suggest mentioning `KillSupervisorsCustomDuty` and how it's used in `common-custom-coordinator-duties` as an example in docs

Reply:
Done